wordpress-mobile/WordPress-Android
# Make tapping on a Zendesk notification show Zendesk tickets list

**NOTE: This PR is an alternate (and better) approach to that of #8043 and is preferred***

Copying the description of the problem here, for completeness of this PR:

> While testing notifications for WPAndroid 10.4 we identified a (what seemed to us broken) flow when tapping a Zendesk notification that only brings you to the `Me` screen in WPAndroid, leaving it up to the user to figure out that they have to tap on `Help & Support` and then `My Tickets`, then finally select the conversation to be able to take it up from there.
> 
> From conversations with @oguzkocer and @aerych it looks like there were many hurdles with this integration, and this was actually a expected behavior as explained in https://github.com/wordpress-mobile/WordPress-Android/issues/7839 and was so described in the tests steps in https://github.com/wordpress-mobile/WordPress-Android/pull/7955
> 

This PR makes tapping on the Zendesk notification bring the "My Tickets" screen to the foreground, in order to complete the desired action (_to look into an HE answer to a question they initiated through `Help & Support -> Contact Us` section of the app_).

BEFORE:
![zendesksnackbarno](https://user-images.githubusercontent.com/6597771/42782828-729f7486-8920-11e8-9dce-fc0809ae84c9.gif)

AFTER:
![zendesknotificationflow](https://user-images.githubusercontent.com/6597771/42838747-fa3da5ea-89d7-11e8-8899-5b48d1ff4270.gif)

To test:
1. initiate a conversation on Zendesk by going to the `Me` tab, then tap `Help & Support` and finally `Contact us`.
2. Send the app to the background (even ideally, just exist the section by tapping `back` twice).
3. have a HE answer your message
4. Observe the push notification is received, and a notification is shown
5. Tap on the notification and see the WP app is opened in the `My Tickets` screen and loads
